### ISSUE TITLE
Criação do manifesto

### ISSUE THREAD

**Issue by jonatasemidio** [socialpullrequest/socialpullrequest#7]
Fala Devs ( @raphamorim  @victorlcampos @ftonato @afonsopacifer   ) ... Gostaria de iniciar com vocês a elaboração do manifesto do SPR... Criei um [MANIFEST.md](https://github.com/socialpullrequest/socialpullrequest/blob/master/MANIFEST.md) para que via PR possamos evoluir o mesmo.

Um exemplo muito bom enviado pelo @victorlcampos foi o [The Zen of Python](https://www.python.org/dev/peps/pep-0020/)

**Comment by victorlcampos** [comment]
1. Como programadores a maneira mais eficiente de doar nosso tempo para sociedade é programando.

**Comment by victorlcampos** [comment]
Eu acredito nisso de verdade, por isso me juntei no projeto.

**Comment by victorlcampos** [comment]
1. Como cidadãos a maneira mais eficiente de doar nosso tempo para sociedade é participando
2. Como programadores a maneira mais eficiente de doar nosso tempo para sociedade é programando.
3. Ajudar fazendo o que gosta é a melhor maneira de continuar ajudando

Nosso sonho grande é ajudar a sociedade da maneira mais eficiente possível, todos realizando o que sabem fazer de melhor.

**Comment by jonatasemidio** [comment]
Exatamente...  A ideia é focar no que as pessoas sabem e gostam de fazer...
Ajudar fazendo o que gosta,  isso me motiva mais ainda.

Algo como faça a diferença com o que você gosta de fazer...  Esta meio
redundante mas é meio que por aí

Em dom, 31 de jan de 2016 11:31, Victor Campos notifications@github.com
escreveu:

**Comment by victorlcampos** [comment]
@raphamorim @ftonato @afonsopacifer ... isso é importante todos participarem =)

**Comment by afonsopacifer** [comment]
Eu acho importante deixar claro nossa missão, além de contribuir, organizar os "eventos" e tentar atrair devs para o movimento e principalmente dar suporte a eles, indicando o que fazer, como ajudar etc...

Acho que muita gente gostaria de ajudar, mas não sabe como, eu mesmo não sei como usar meu código para ajudar as pessoas. Acho que temos que fazer essa ponte mesmo, entre os Devs e os projetos que precisão de um UP.

Sei lah... Essas coisas são importantes estarem claras..

**Comment by victorlcampos** [comment]
Concordo 100%...
Vc acha que consegue resumir isso em um texto bonitinho?

**Comment by afonsopacifer** [comment]
Acho que sim, amanha eu vejo isso, e posto aqui :)

**Comment by jonatasemidio** [comment]
Inseri alguns [#itens](https://github.com/socialpullrequest/socialpullrequest/commit/ba047fe35c6b00a5e3d1a536cedf2d8ac92240dc) meu e que foram trocados por email... Assim que der deem uma olhada por favor.
